app/src/main/java/org/wikipedia/edit/EditSectionActivity.kt

```diff
@@ -418,23 +420,24 @@ class EditSectionActivity : BaseActivity(), ThemeChooserDialog.Callback, EditPre
      */
     private fun handleEditingException(caught: MwException) {
         val code = caught.title
-
-        // In the case of certain AbuseFilter responses, they are sent as a code, instead of a
-        // fully parsed response. We need to make one more API call to get the parsed message:
-        if (code.startsWith("abusefilter-") && caught.message.contains("abusefilter-") && caught.message.length < 100) {
-            disposables.add(ServiceFactory.get(pageTitle.wikiSite).parsePage("MediaWiki:" + StringUtil.sanitizeAbuseFilterCode(caught.message))
-                    .subscribeOn(Schedulers.io())
-                    .observeOn(AndroidSchedulers.mainThread())
-                    .subscribe({ response: MwParseResponse -> showError(MwException(MwServiceError(code, response.text))) }) { showError(it) })
-        } else if ("editconflict" == code) {
-            MaterialAlertDialogBuilder(this@EditSectionActivity)
+        lifecycleScope.launch(CoroutineExceptionHandler { _, t ->
+            showError(caught)
```

Comment:
Do we need to show `t` from `CoroutineExceptionHandler` or the `caught` from the method parameter?

Reply:
t! done!